package/MDAnalysis/lib/include/calc_distances.h

```diff
@@ -531,54 +531,67 @@ static void _calc_angle_triclinic(coordinate* atom1, coordinate* atom2,
   }
 }
 
+static double _calc_dihedral_angle(double* va, double* vb, double* vc)
```

Comment:
I think there might be a minor gain in performance if this function already takes the pointer to the result array slot, and write directly there instead of returning a double.